test/e2e/common/downwardapi.go

```diff
@@ -284,6 +284,7 @@ var _ = SIGNodeDescribe("Downward API", func() {
 	})
 })
 
+// TODO(#98326): Is this sig-node or sig-storage test?
```

Comment:
@dchen1107 @derekwaynecarr - can you please advise on it?

[it's also possible that split by SIG maybe at the level of individual tests here]

Reply:
It is a joint project initiated by both SIG Storage and SIG Node, but driven by SIG storage later. Mark it for SIG Storage please.

Reply:
Thanks - done